.gitmodules

```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/aquynh/capstone.git
 [submodule "external/crypto-algorithms"]
 	path = external/crypto-algorithms
-	url = https://github.com/B-Con/crypto-algorithms
+	url = https://github.com/KorewaWatchful/crypto-algorithms
```

Comment:
wrong, need using vita3k git 🤔

Reply:
not anymore :D

Reply:
rebase on master, you have uslkess commit almready pressent on master, don't have sense

Reply:
If we are about to change something that module, I think we should fork that repo to `Vita3K` and create a PR to that repo first.

Reply:
Why do you fork that module to change `sha256_init` to `sha256_init_one`? Is there any naming collision?

Reply:
yes there is

Reply:
if you want to discuss, please go to discord. This is a draft PR as you can see.